From 923dda8c4fe637a190966c8b9a477b1a5f73497b Mon Sep 17 00:00:00 2001
From: Chandrakala Chavva <cchavva@caviumnetworks.com>
Date: Tue, 22 Jul 2014 18:05:50 -0700
Subject: [PATCH 914/974] MIPS:OCTEON: Increase the load address

Bug #15939:
The size of BAR0 is increased on cn73xx from 32KB to 8MB.

Signed-off-by: Chandrakala Chavva <cchavva@caviumnetworks.com>
[Original patch taken from Cavium SDK 3.1.2-568]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/cavium-octeon/Platform | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/Platform b/arch/mips/cavium-octeon/Platform
index 1069c2f..4105c82 100644
--- a/arch/mips/cavium-octeon/Platform
+++ b/arch/mips/cavium-octeon/Platform
@@ -8,6 +8,6 @@ cflags-$(CONFIG_CAVIUM_OCTEON_SOC)	+=				\
 ifdef CONFIG_MAPPED_KERNEL
   load-$(CONFIG_CAVIUM_OCTEON_SOC)    = 0xffffffffc0000000
 else
-  load-$(CONFIG_CAVIUM_OCTEON_SOC)    = 0xffffffff80100000
+  load-$(CONFIG_CAVIUM_OCTEON_SOC)    = 0xffffffff80800000
 endif
 
-- 
2.6.2

